src/ClientBuilderInterface.php

```diff
@@ -22,11 +22,18 @@
     /**
      * Creates a new instance of this builder.
      *
-     * @param array $options The client options
+     * @param array $options The client options, in naked array form; it will be used to construct {@see Options}
```

Comment:
I don't think that it's useful to know that the array will be used to construct an object instance, I would remove this part of the description